### DISCUSSION TITLE
Background-color applied to table rounded corners

### DISCUSSION THREAD

**Discussion by andcl** [dompdf/dompdf#2605] [General]
First of all, thanks @bsweeney for this incredible project. 
Based on [this question](https://github.com/dompdf/dompdf/issues/2013), I have another one:

Say we have a rounded-corners table like the one of the preceding question. When I try to give a `background-color`, e.g., to the table `<th>`, I get that the area resulting from the difference between the whole cell and the `rounded` one also gets the background color applied (i.e. the whole rectangular cell is colored). Is there an easy workaround for this situation?

Thanks in advance.

**Comment by bsweeney** [comment]
Sorry, do you have a sample I'm having trouble visualizing.

**Comment by Mellthas** [comment]
Is this #2428? If so, #2606 will fix the issue.

**Reply by andcl** [reply]
Yes, after asking I realized that #2428 was about the same problem I was facing. Thanks.
